core/internal/filestream/updatestats.go

```diff
@@ -45,6 +45,13 @@ func (u *StatsUpdate) Apply(ctx UpdateContext) error {
 	}
 
 	line, err := simplejsonext.Marshal(row)
+
+	// Override the default max line length if the user has set a custom value.
+	maxLineBytes := ctx.Settings.GetFileStreamMaxLineBytes()
```

Comment:
should we also allow for negative values as a way to specify that we would like to remove this limit completely?

Reply:
nah, just let them use a really big value

Reply:
no line would overflow int32, i presume

Reply:
ok for now i'll treat it as if no value was set

Reply:
i was thinking the same i can use int64 but max int32 is so huge already, what do you think?